crates/runtime/src/component/logging.rs

```diff
@@ -47,6 +47,9 @@ impl<H: Handler> logging::Host for Ctx<H> {
         message: String,
     ) -> anyhow::Result<()> {
         self.attach_parent_context();
+        if level == logging::Level::Error {
+            eprintln!("Error log dropped: {} - {}", context, message);
+        }
```

Comment:
I see the thinking here, but in this case this is actually just the component logging and an Error level log is not a dropped log. So this should be removed